lib/rubygems/commands/info_command.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+
+require 'rubygems/command'
+require 'rubygems/commands/query_command'
+
+class Gem::Commands::InfoCommand < Gem::Commands::QueryCommand
+  def initialize
+    super "info", "Show information for the given gem"
+
+    remove_option('--name-matches')
+    remove_option('-d')
+
+    defaults[:details] = true
+    defaults[:exact] = true
+  end
+
+  def description # :NODOC:
```

Comment:
These are in this project often written as `:nodoc:` - is there a specific reason why these are upper-cased?